spec/genre_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../lib/classes/genre'
+require_relative '../lib/classes/item'
+require 'json'
+
+describe Genre do
+  context 'When testing creation of genre instance' do
+    it 'Should create a genre when we call the class constructor' do
+      new_genre = Genre.new('Test')
+      expect(new_genre.name).to eq 'Test'
+    end
+  end
+  context 'When testing genre methods' do
+    it 'Should add a new item to the items array' do
+      new_genre = Genre.new('Test')
+      item = Item.new(2)
+      new_genre.add_item(item)
+      expect(new_genre.items.length).to eq 1
+    end
+  end
```

Comment:
- [ ] @Team great work adding tests for the genre class however you forgot to add tests for these methods: to_s,to_json, and self.from_json

Reply:
- [ ] @Team I noted you have not added tests for the item class kindly consider adding tests for this class